Example/Tests/Helpers/SpyTracker.swift

```diff
@@ -9,32 +9,44 @@
 import Foundation
 
 class SpyTracker {
-    var spies: [String: [String: String]?] = [:]
+    var spies: [String: [[String: String]?]] = [:]
```

Comment:
Previously this was only holding the last call to a function. In the case where a spy is called multiple times, previous calls were lost. I extended it to keep all calls to that expectations can also verify previous calls.